### ISSUE TITLE
User Profile

### ISSUE THREAD

**Issue by jmt4** [thethomp/lammergeyer#16]
Create a user profile page for information updating.

Not sure where we are gonna go with phone numbers. An idea is that as part of the registration form, they can register with phone, or email or both.

Or and also, we can create a profile page and update a phone number there.

Thoughts?

**Comment by thethomp** [comment]
Yep, we should have phone number and email fields on the user registration page, though phone number won't have to be required (email will). If people don't register their phone number when they make an account, they should have the option to add it later via an "Update Profile" page or something.
